src/main/drivers/gyro_sync.c

```diff
@@ -14,13 +14,28 @@
 #include "accgyro.h"
 #include "gyro_sync.h"
 
+#include "common/axis.h"
+
+#include "sensors/gyro.h"
+
 static uint8_t mpuDividerDrops;
+static volatile bool pidDataReady;
+static uint8_t pidProcessDenom, pidProcessCountDown;
 
-bool gyroSyncCheckUpdate(const gyro_t *gyro)
+bool isGyroDataReady(void)
```

Comment:
Another misnomer. This is not about whether the gyro data is ready - the gyro data may have been ready for some time. This is about whether the PID controller should run. So it should be called `isPidScheduledToRun`.

That's a name clash with the variable above, which isn't surprising since they are about the same thing. So let's call the variable `pidScheduledToRun`.